Header header

int8 pointId
float64 northing
float64 easting
float64 height
float64 hz_angle
float64 v_angle
float64 sd
int8 EDM_kind
int8 EDM_mode
string date
time time_
time sensor_time